product-types/product-types.json

```diff
@@ -1,11 +1,7 @@
-[
```

Comment:
the app expects an array of product types, so the JSON parser fails with an object